cdap-ui/app/cdap/services/PipelineMetricsStore/index.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {createStore} from 'redux';
+import {defaultAction, composeEnhancers} from 'services/helpers';
+
+const DEFAULT_METRICS_STATE = {
+  metrics: [],
+  logsMetrics: {},
+  // FIXME: There are probably better places to store these info, but adding them here
+  // for now as the old metrics store had these info. Can come back and move these later
+  metricsTabActive: false,
```

Comment:
Right now this is not valid right? In the old UI we had tabbed view and we needed to check which tab we were in. Is this still valid? Do we need this?

Reply:
This is referring to the Metrics tab in the node config modal. If the user clicks on a node's metrics to open up the node config modal (as opposed to clicking on the 'Properties' button), then we want to show the Metrics tab.